docs/bokeh/source/conf.py

```diff
@@ -169,6 +169,12 @@
 
 html_title = f"{project} {version} Documentation"
 
+# avoid CORS error on local docs build for the switcher.json file
+if 'dev' in version:
```

Comment:
Here we need some better logic to decide between a local and a online development version.

Reply:
I'm not too concerned about the `conf.py` contents. Anything quick and easy that works will do

Reply:
How is the process for an offical release? AFAIK the online version for `dev-3.5` would get the wrong `json_url` which is wrong. I just want to make sure that I don't break the development documentation.

Reply:
Ah, I missed that this was the concern. Some ideas:

* usually local docs are / should be build with `BOKEH_DOCS_CDN="local"` set as described [here](https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#build-bokeh-s-documentation). I am not sure everyone reliably sets this when doing local dev, though
* actual commands for release scenario are here:
   https://github.com/bokeh/bokeh/blob/d08906b168341db5a4b0fb0b47a1372cd7f0f369/release/build.py#L72

  We could possibly cue off something in the env vars there, or if need be we could add some new make target to differentiate between local and release buid

Reply:
If I add `BOKEH_DOCS_RELEASE` to

https://github.com/bokeh/bokeh/blob/d08906b168341db5a4b0fb0b47a1372cd7f0f369/release/build.py#L72

then I can check in the `conf.py` for the existance of `BOKEH_DOCS_RELEASE` and if it exists, it is a release, am I right?

I pushed a commit with this change and I belive the issue is solved. @bryevdv Could you please leave a comment?

Reply:
@mosc9575 I'm just a little reticent to add a third env var in the release call, it's already pretty clunky with two. Thinking a bit more on it, I can't really think of any reason anyone would ever set `BOKEH_DOCS_VERSION` locally, except in the very rare case of manually rebuilding and redeploying docs, so why don't we just condition on the presence or absence of that? (and In fact if we are in that rare case, conditioning off that will yield the correct behavior more reliably)